src/ipc/print.ts

```diff
@@ -43,6 +49,10 @@ async function printData({
     lprOptions.push('InputSlot=' + paperSource)
   }
 
+  if (typeof copies !== 'undefined') {
+    lprOptions.push('-#', copies.toString())
```

Comment:
should we check that copies is an integer (as opposed to having fractional copies)?